src/ml/neural_net/tf_compute_context.cpp

```diff
@@ -0,0 +1,63 @@
+/* Copyright © 2018 Apple Inc. All rights reserved.
+ *
+ * Use of this source code is governed by a BSD-3-clause license that can
+ * be found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
+*/
+
+#include <iostream>
+#include <vector>
+
+#include <ml/neural_net/tf_compute_context.hpp>
```

Comment:
Nit: In a .cpp file, let's always the corresponding .hpp file first, in its own grouping.